Kitodo/pom.xml

```diff
@@ -233,6 +233,7 @@
         <dependency>
             <groupId>org.xmlunit</groupId>
             <artifactId>xmlunit-matchers</artifactId>
+            <scope>test</scope>
```

Comment:
Not needed as defined in parent as test scope.

Reply:
You add the right scope in your current pull request and as the scopes are passed to the sub modules you did not define it again. As already written in my post above.